src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -36,33 +36,36 @@ class NotebookCard extends Component {
     const { namespace, name, updated, listView, wsName, onRename, onCopy, onDelete, canCompute, canWrite } = this.props
 
     const notebookMenu = canWrite && h(PopupTrigger, {
-      position: 'bottom',
+      position: 'right',
       closeOnClick: true,
       content: h(Fragment, [
         h(MenuButton, {
           onClick: () => onRename()
-        }, ['Rename']),
+        }, [icon('renameIcon', { size: 15, style: { margin: '0 .25rem 0 0' } }), 'Rename']),
         h(MenuButton, {
           onClick: () => onCopy()
-        }, ['Duplicate']),
+        }, [icon('copy', { size: 15, style: { margin: '0 .25rem 0 0' } }), 'Clone']),
         h(MenuButton, {
           onClick: () => onDelete()
-        }, ['Delete'])
+        }, [icon('trash', { size: 15, style: { margin: '0 .25rem 0 0' } }), 'Delete'])
       ])
     }, [
       h(Clickable, {
         onClick: e => e.preventDefault(),
-        style: { marginLeft: '1rem', cursor: 'pointer' }, focus: 'hover'
-      }, [icon('ellipsis-vertical', { size: 18 })])
+        style: {
+          cursor: 'pointer', color: colors.blue[0]
+        },
+        focus: 'hover',
+        hover: { opacity: 0.5 }
```

Comment:
nit: Rather than using opacity, we could just choose the next lighter blue from our palette.